dione-spark/src/test/scala/com/paypal/dione/spark/index/avro/TestOneLineInBlock.scala

```diff
@@ -33,6 +33,9 @@ object TestOneLineInBlock extends SparkCleanTestDB {
       s"partitioned by (dt string) stored as avro")
     spark.sql(s"insert overwrite table t3 partition (dt='2018-10-04') select * from t")
 
+    // ensure one row per block
+    assert(spark.table("t3").filter($"data_sub_offset" > 0).count == 0)
```

Comment:
t3 is the data table, not the index table..

Reply:
Fail attempt to PR via github editor :)
fixed